src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -256,23 +256,42 @@ private static void WriteLinkerScriptFile(string exportsFile, List<string> expor
 
     private void WriteAutoInitializationFromTemplate()
     {
+        string logError = GenerateLog("error");
+        string logInfo = GenerateLog("info");
         File.WriteAllText(Path.Combine(OutputDirectory, "autoinit.c"),
             Utils.GetEmbeddedResource("autoinit.c")
                 .Replace("%ASSEMBLIES_LOCATION%", AssembliesLocation ?? "DOTNET_ASSETS_PATH")
+                .Replace("%LOG_ERROR%", logError)
+                .Replace("%LOG_INFO%", logInfo)
                 .Replace("%RUNTIME_IDENTIFIER%", RuntimeIdentifier));
     }
 
+    private string GenerateLog(string level)
```

Comment:
My feeling is that it's better to `#ifdef` these in autoinit.

Reply:
Used `#ifdef` instead and passed `-DHOST_ANDROID` into the CMakeLists `if(TARGETS_ANDROID)` block